src/test/rustdoc/async-fn.rs

```diff
@@ -1,12 +1,24 @@
 // edition:2018
-// compile-flags:-Z unstable-options
-
-// FIXME: once `--edition` is stable in rustdoc, remove that `compile-flags` directive
 
 #![feature(async_await, futures_api)]
 
+// @has async_fn/fn.foo.html '//pre[@class="rust fn"]' 'pub async fn foo() -> Option<S>'
+pub async fn foo() -> Option<S> {
```

Comment:
I thought that `S` was a type parameter because the name was only one letter. Can you make it at least two letters so people won't struggle on it in the future (haha) please?